rpa-manager/modules/ROOT/pages/processautomation-manage.adoc

```diff
@@ -242,8 +250,51 @@ Open the xref:processoperations-financeanalysis.adoc[*Finance Analysis* view] to
 
 Once it's ready, transition the project to the next phase of its life-cycle. You can always transition back to change the project's design or implementation.
 
+In the **Design** phase, you can do the following:
+
+* Create or change a process model in RPA Manager.
++
+Already implemented activities don’t lose their implementation if you don’t delete them. 
+* Transfer the automation containing the new or changed models to the Build Phase.
+
+In the **Build** phase, in RPA Builder, you can do the following:
+
+* Download the model from RPA Manager.
+* Change the model.
+* Implement the activities.
+* If you want to version workflows, create and reuse activities in the Activity Library.
+* Upload the changed and implemented model to the repository in RPA Manager. 
++
+There is always only one version of a process in the repository in RPA Manager. There might be another local version in RPA Builder. If you accidentally deleted an already implemented Activity, you might still be able to find it in RPA Builder - as long as you do not overwrite the model by downloading.
+* Transfer the automation back to the Design phase.
++
+Already implemented activities don’t lose their implementation if you don’t delete them. 
+* Transfer the automation on to the Test phase.
```

Comment:
```suggestion
* Transfer the automation to the test phase.
```

Reply:
Global: should "transfer" be "transition"? I think "transition" is more commonly used in this case (transition from one phase to another).

Reply:
Yes. Changed all. But it seems to be a very technical expert word to me, ,which I would personally prefer, because it's more precise. Is it also used in daily speak?

Reply:
The other thing is that users might use different terms to search for a topic. Using only one term (which is preferred in the style guide) might lead to users not finding what they search for. (This could be solved with a better search algorithm)

Reply:
I think transition is often used for state changes. But you bring up a good point and something we've been thinking about - metadata for SEO.